tests/test_multiannotator.py

```diff
@@ -0,0 +1,187 @@
+
+    assert isinstance(labels_wide, pd.DataFrame)
+    # TODO: might add a test to see if the wide dataframe actually contains the right info
+
+
+# TODO: update this test to work with NaN values - randomly deleting data will make assertion invalid
+def test_get_worst_class():
+    labels = data["complete_labels"][0]  # only testing on first column
+    pred_probs = data["pred_probs"]
+
+    # Assert that the worst class index should be the class with the highest noise
+    assert dataset._get_worst_class(labels, pred_probs) == data["noise_matrix"].diagonal().argmax()
+
+
+def test_label_quality_scores_multiannotator():
+    labels = data["labels"]
+    pred_probs = data["pred_probs"]
+
+    lqs_multiannotator = get_label_quality_multiannotator(labels, pred_probs)
+    assert isinstance(lqs_multiannotator, pd.DataFrame)
+
+    # test verbose=False
+    lqs_multiannotator = get_label_quality_multiannotator(labels, pred_probs, verbose=False)
+    assert isinstance(lqs_multiannotator, pd.DataFrame)
+
+    # test passing a list into consensus_method
+    # TODO: change list items after adding more options
+    lqs_multiannotator = get_label_quality_multiannotator(
+        labels, pred_probs, consensus_method=["majority", "majority"]
+    )
+
+    # test different quality_methods
+    # also testing passing labels as np.ndarray
+    lqs_multiannotator = get_label_quality_multiannotator(
+        np.array(labels), pred_probs, quality_method="agreement"
+    )
+
+    # test returning annotator stats
+    lqs_annotatorstats = get_label_quality_multiannotator(
+        labels, pred_probs, return_annotator_stats=True
+    )
+    assert isinstance(lqs_annotatorstats, tuple)
+    assert isinstance(lqs_annotatorstats[0], pd.DataFrame)
+    assert isinstance(lqs_annotatorstats[1], pd.DataFrame)
+
+    # test error catching when labels_multiannotator has NaN columns
+    labels_NA = deepcopy(labels)
+    labels_NA[0] = pd.NA
+
+    try:
+        lqs_annotatorstats = get_label_quality_multiannotator(
+            labels_NA, pred_probs, return_annotator_stats=True
+        )
+    except ValueError as e:
+        assert "cannot have columns with all NaN" in str(e)
+
+
+@pytest.mark.filterwarnings("ignore::UserWarning")
+def test_get_consensus_label():
+    labels = data["labels"]
+
+    # getting consensus labels without pred_probs
+    consensus_label = get_consensus_label(labels)
+
+    # making synthetic data to test tiebreaks of get_consensus_label
+    # also testing pssing labels as np.ndarray
+    labels_tiebreaks = np.array([[1, 2, 0], [1, 1, 0], [1, 0, 0], [2, 2, 2], [1, 2, 0], [1, 2, 0]])
+    pred_probs_tiebreaks = np.array(
+        [
+            [0.4, 0.4, 0.2],
+            [0.3, 0.6, 0.1],
+            [0.75, 0.2, 0.05],
+            [0.1, 0.4, 0.5],
+            [0.2, 0.4, 0.4],
+            [0.2, 0.4, 0.4],
+        ]
+    )
+
+    consensus_label = get_consensus_label(labels_tiebreaks, pred_probs_tiebreaks)
```

Comment:
Great test!  Can we add 1-2 more edge-case tests like this?  In particular for sparse setting where most examples only have 1 annotation, there are some annotators that don't overlap with others, etc.